docs/examples/EKS.md

```diff
@@ -202,4 +202,30 @@ spec:
   eks:
     configuration:
       image: latest
-```
\ No newline at end of file
+```
+
+
+## SSM-based versioned AMI
+
+It might not be acceptable to pin your `InstanceGroup` to the latest AMI. If you want to pin to a specific version,
+you can specify the relevant version information using syntax like `ssm://version`. An example for the EKS-optimized Linux AMI might look like:
+
+```yaml
+apiVersion: instancemgr.keikoproj.io/v1alpha1
+kind: InstanceGroup
+metadata:
+  name: hello-world
+  namespace: instance-manager
+  annotations:
+    instancemgr.keikoproj.io/os-family: "amazonlinux2" #Default if not provided
+spec:
+  strategy: <...>
+  provisioner: eks
+  eks:
+    configuration:
+      image: ssm://amazon-eks-node-1.18-v20220226 # resolves to value in /aws/service/eks/optimized-ami/1.18/amazon-linux-2/amazon-eks-node-1.18-v20220226/image_id
```

Comment:
So the difference essentially between `latest` and `ssm://amazon-eks-node-1.18-v20220226` is that you get control over the `/amazon-eks-node-1.18-v20220226/` section of the URI, where `latest` would probably get a 1.22 or newer version in that case?
Should we also create a bug for the `latest` case? it seems latest should also pin to the cluster version

Reply:
Also what we'll get by fixing the `latest` pinning case is once the cluster is upgraded to a newer version, it will immediately trigger a node rotation

Reply:
Latest is already looking up the cluster k8s version, and selecting the appropriate AMI.

Reply:
Oh OK, so essentially why we want this feature is to be able to control `-v20220226` ?

Reply:
Or if you don't want your nodes to move with cluster

Reply:
Yes, the former.